styles.css

```diff
@@ -382,8 +435,372 @@ textarea {
+  }
+
+  .nav-menu.active {
+    left: 0;
+    margin-top: 20px;
+    background: #3c3a39;
+    box-shadow: 0 48px 48px rgba(37, 47, 137, 0.08);
+  }
+
+  .nav-menu.active .nav-item {
+    font-weight: 700;
+    line-height: 44px;
+    color: #fff5e1;
+    border-bottom: 1px solid #6f6c6b;
+    padding: 1rem;
+    margin: 0 1.5rem;
+  }
+
+  .icon-mobile.active .bar:nth-of-type(1) {
+    transform: translateY(6.3px) rotate(45deg);
+  }
+
+  .icon-mobile.active .bar:nth-of-type(2) {
+    opacity: 0;
+  }
+
+  .icon-mobile.active .bar:nth-of-type(3) {
+    transform: translateY(-6.3px) rotate(-45deg);
+  }
+}
+
+/* Start styling desktop version */
+@media (min-width: 768px) {
+  .mobile-home {
+    background-image: none;
+    display: flex;
+    width: 75.8%;
+    font-size: 100%;
+    margin-right: 14%;
+    margin-left: 12%;
+    flex-direction: column;
+    justify-content: center;
+    align-items: center;
+    padding: 5%;
+    letter-spacing: 10%;
+  }
+
+  body {
+    background-image: url(assets/images/Header-llustration-desktop.svg);
+    background-repeat: no-repeat;
+    background-size: contain;
+    margin: 0;
+    padding: 0;
+  }
+
+  .header {
+    display: flex;
+    flex-direction: row;
+    gap: 30px;
+    font-size: 100%;
+    justify-content: center;
+    align-items: baseline;
+    margin: 0% auto;
+  }
+
+  .header li a {
+    font-family: "Inter", sans-serif;
+    font-style: normal;
+    font-size: 85%;
+    font-weight: 700;
+    cursor: pointer;
+    color: #172b4d;
+  }
+
+  .logo {
+    font-family: "Inter", Helvetica, Arial, sans-serif;
+    font-size: 1.125rem;
+    transform: rotate(360deg);
+    transition: 1s;
+    margin-right: 26%;
+  }
+
+  .nav-menu {
+    list-style-type: none;
+    display: flex;
+    align-items: center;
+    gap: 0.5rem;
+    font-weight: 600;
+    font-size: 1rem;
+    line-height: 1.25rem;
+    margin-right: 22rem;
+  }
+
+  .nav-item {
+    padding: 0.625rem 0.75rem;
+  }
+
+  .icon-mobile {
+    display: none;
+  }
+
+  .main-container {
+    display: flex;
+    flex-direction: column;
+    text-align: center;
+    margin: 1%;
+    width: 98%;
+    align-items: center;
+    justify-content: center;
+  }
+
+  .primary {
+    margin: -4% 1% 1% 0%;
+    font-weight: 650;
+  }
+
+  .half-primary {
+    font-weight: 650;
+  }
+
+  .supporting-text {
+    margin: 1% 19% 2% 20%;
+    font-weight: 450;
+    line-height: 120%;
+    text-align: center;
+  }
+
+  .social-media-icons li {
+    margin: 2% 1%;
+  }
+
+  .action {
+    color: #505f79;
+  }
+
+  .icon-chevron-down-larger {
+    margin-top: 5%;
+  }
+
+  .primary-text {
+    font-weight: 650;
+    font-size: 340%;
+    text-align: center;
+    width: 180%;
+    margin: 0%;
+    text-decoration: none;
+  }
+
+  /* My Recent Works */
+
+  .card-work-row1 {
+    display: grid;
+    grid-template-columns: auto auto auto;
+    grid-template-rows: auto auto;
+    justify-content: space-evenly;
+    justify-items: stretch;
+  }
```

Comment:
- [ ] Kindly make sure you stick to the original design as much as possible. your typo(fonts), and spacing(margin. padding) are supposed to be similar to the ones on the original template. Kindly make sure the size of your cards and position on the UI are identical to the design in that way your page will have a great look and feel.

Your design :point_down: 
![Screenshot from 2023-06-09 14-58-55](https://github.com/yesuf023/Portfolio/assets/97100091/7a49d259-83c6-4ce3-aeb1-38e9d66fa093)

### Original template :point_down: 
![image](https://github.com/yesuf023/Portfolio/assets/97100091/9ff2f27f-637f-4882-a0fd-7e750b072738)